### PR TITLE
Add popup to <Mentoring.Queue />

### PR DESCRIPTION
**Pull request by kntsoriano**
@ErikSchierboom This is the new PR with Popper added to the right component. I've also applied your suggestions here. Thanks :)

### CODE REVIEW

File: app/javascript/components/mentoring/Queue/Solution.jsx [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import React from 'react'
```
Comment (ErikSchierboom) [inline app/javascript/components/mentoring/Queue/Solution.jsx]: There is now a double React import which leads to a runtime error. Could you remove the first line?

```suggestion
```

**Review by ErikSchierboom — COMMENTED** [review]
>  I've also applied your suggestions here

I don't seem them applied in this PR? Am I missing something?

**Comment by kntsoriano** [comment]
@ErikSchierboom I've set `null` to `false`.

File: app/javascript/components/mentoring/Queue/Solution.jsx [hunk 2]
```diff
@@ -13,29 +14,53 @@ export function Solution({
...
-    </tr>
+    <>
+      <tr
+        ref={setReferenceElement}
+        onMouseEnter={() => {
+          setShowPopper(true)
+        }}
+        onMouseLeave={() => {
+          setShowPopper(false)
+        }}
```
Comment (ErikSchierboom) [inline app/javascript/components/mentoring/Queue/Solution.jsx]:
```suggestion
        onMouseEnter={() => setShowPopper(true)}
        onMouseLeave={() => setShowPopper(false)}
```

**Review by ErikSchierboom — APPROVED** [review]

**Comment by iHiD** [comment]
Tis red.

**Pull request merged**